### PR TITLE
feat:add single party func abs

### PR DESCRIPTION
**Pull request by cs1317**
在SCQL中实现单方算子abs
https://github.com/secretflow/scql/issues/112

### CODE REVIEW

**Comment by tongke6** [comment]
@cs1317 please sign the CLA

**Comment by cs1317** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by cs1317** [comment]
I have read the CLA Document and I hereby sign the CLA

File: pkg/expression/expression_to_stmt.go [hunk 1]
```diff
@@ -123,6 +123,8 @@ func (c ExprConverter) convertScalarFunction(dialect format.Dialect, expr *Scala
 		return &ast.FuncCallExpr{FnName: model.NewCIStr(ast.If), Args: children}, nil
 	case ast.Cos:
 		return &ast.FuncCallExpr{FnName: model.NewCIStr(expr.FuncName.L), Args: children}, nil
+	case ast.Abs:
```
Comment (tongke6) [inline pkg/expression/expression_to_stmt.go:126]: DRY (Don't Repeat Yourself)

126-127 和 前两行代码可以写在一个 case 里

File: pkg/expression/expression_to_stmt.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func (c ExprConverter) convertScalarFunction(dialect format.Dialect, expr *Scala
 		return &ast.FuncCallExpr{FnName: model.NewCIStr(dialect.GetSpecialFuncName(ast.Ifnull)), Args: children}, nil
 	case ast.If:
 		return &ast.FuncCallExpr{FnName: model.NewCIStr(ast.If), Args: children}, nil
-	case ast.Cos:
+	case ast.Cos,ast.Abs:
```
Comment (tongke6) [inline pkg/expression/expression_to_stmt.go:124]: please format the code

**Review by tongke6 — APPROVED** [review]
LGTM

**Comment by cs1317** [comment]
recheck

**Comment by tongke6** [comment]
image

@cs1317 你好，CLA check 一直没过，应该和第一个 commit 的作者信息不对有关系，麻烦看下如何解决
image

**Pull request closed without merging**